### PR TITLE
boot.janet docstrings (part 2)

### PR DESCRIPTION
**Pull request by uvtc**
Added some backticks around code in docstrings to distinguish them from prose.

Also some light editing.

### CODE REVIEW

File: src/boot/boot.janet [hunk 1]
```diff
@@ -1193,8 +1193,8 @@
   "Bound to the name of the currently compiling file.")
 
 (defmacro tracev
-  `Print a value and a description of the form that produced that value to
-  stderr. Evaluates to x.`
+  `Print to stderr a value and a description of the form that produced that value.
+  Evaluates to x.`
```
Comment (pepe) [inline src/boot/boot.janet:1197]: I guess x could be in `` too?

File: src/boot/boot.janet [hunk 2]
```diff
@@ -1235,11 +1235,11 @@
   (reduce fop x forms))
 
 (defmacro -?>
-  `Short circuit threading macro. Inserts x as the second value in the first form
-  in forms, and inserts the modified first form into the second form
+  ``Short circuit threading macro. Inserts x as the second value in the first form
```
Comment (pepe) [inline src/boot/boot.janet:1238]: Again, x could be in ``?

File: src/boot/boot.janet [hunk 3]
```diff
@@ -1251,11 +1251,11 @@
   (reduce fop x forms))
 
 (defmacro -?>>
-  `Short circuit threading macro. Inserts x as the last value in the first form
-  in forms, and inserts the modified first form into the second form
+  ``Short circuit threading macro. Inserts x as the last value in the first form
```
Comment (pepe) [inline src/boot/boot.janet:1254]: x in ``?

File: src/boot/boot.janet [hunk 4]
```diff
@@ -1300,14 +1300,14 @@
...
+  "Similar to `postwalk`, but do pre-order traversal."
   [f form]
   (walk (fn [x] (prewalk f x)) (f form)))
 
 (defmacro as->
-  `Thread forms together, replacing as in forms with the value
-  of the previous form. The first for is the value x. Returns the
-  last value.`
+  ``Thread forms together, replacing `as` in `forms` with the value
+  of the previous form. The first form is the value x. Returns the
```
Comment (pepe) [inline src/boot/boot.janet:1309]: x in ``?

File: src/boot/boot.janet [hunk 5]
```diff
@@ -1317,10 +1317,10 @@
   prev)
 
 (defmacro as?->
-  `Thread forms together, replacing as in forms with the value
-  of the previous form. The first for is the value x. If any
+  ``Thread forms together, replacing `as` in `forms` with the value
+  of the previous form. The first form is the value x. If any
```
Comment (pepe) [inline src/boot/boot.janet:1321]: x in ``?

File: src/boot/boot.janet [hunk 6]
```diff
@@ -1365,17 +1365,16 @@
...
   [ind]
   (var res true)
   (loop [x :in ind :while res]
     (if x nil (set res x)))
   res)
 
 (defn any?
-  `Returns the first truthy value in ind, otherwise nil.
-  falsey value.`
+  ``Returns the first truthy value in `ind`, otherwise nil.``
```
Comment (pepe) [inline src/boot/boot.janet:1377]: Shouldn't nil be too in ``?

**Review by pepe — CHANGES_REQUESTED** [review]
I am not sure, if `x`s make sense?

**Comment by uvtc** [comment]
"x" shows up so often, and is so self-explanatory, that I usually don't put it into backticks (unless it's right next to some other parameter that was put into backticks, and it would look weird if the x were *not* in backticks).

I try to have a light touch here and not be slavishly consistent in cases where the docstring is already clear as-is.

My primary goal is to clarify the docstring prose, especially in cases where a parameter name or other function name (code) can be mistaken for prose.

**Review by pepe — APPROVED** [review]
LGTM

**Comment by pepe** [comment]
Thank you for the explanation, it makes sense.

**Pull request merged**